installers/sublime text/installer.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+# Sublime Text 3 Install 
+#
+# No need to download this script, just run it on your terminal:
+#
+#     https://installer.to/sublimetext | bash
+
+
+# Detect the architecture
+if [[ "$(uname -m)" = "x86_64" ]]; then
+  ARCHITECTURE="x64"
+else
+  ARCHITECTURE="x32"
```

Comment:
I think we should check if the architecture is 32 or not too, because we can't assume it's 32 if it's not 64, there are other types like `arm`, `mips`

Reply:
I put an elif instead of else so it should be alright now :)